src/legacy/wildwest.rs

```diff
@@ -138,6 +139,7 @@ impl Default for GlobalConfig {
             dest: "./".to_owned(),
             layouts: "_layouts".to_owned(),
             drafts: "_drafts".to_owned(),
+            data: "_data".to_owned(),
```

Comment:
Something I didn't think about: is there a use case for this to be configurable?

For example, something I want to do is eventually remove the option for configuring `layouts` because I can't think of a justification for it.

I'm looking at tagging a release and changing this before then means I don't have to break compatibility.

Reply:
For data (unlike sass) I could see the point being made, that different tools have different names so this could ease import/testing it with an existing repo. But I am also fine with removing this, if you think this causes confusion.